proto/chainlink/v1beta/tx.proto

```diff
@@ -159,6 +161,26 @@ message MsgRequestNewRound {
   bytes signer = 3 [(gogoproto.casttype) = "github.com/cosmos/cosmos-sdk/types.AccAddress"];
 }
 
+// MsgAccount is the type defined for account associated with the Chainlink node
+message MsgAccount {
+  // submitter - associated cosmos account address
+  bytes submitter  = 1 [(gogoproto.casttype) = "github.com/cosmos/cosmos-sdk/types.AccAddress"];
```

Comment:
hmm but what do you mean..? it would be the same as the one in `feed data`! the submitter of the feed data should correspond to a `MsgAccount`, that way they can be paid!

Let me know if that is still confusing and what you would like to see it changed to!

Reply:
I got what you mean now, nvm. I will mark it as resolved. thanks